examples/rvc-app/rvc-common/rvc-app.matter

```diff
@@ -0,0 +1,1484 @@
+    callback attribute networks;
+    ram      attribute scanMaxTimeSeconds;
+    ram      attribute connectMaxTimeSeconds;
+    ram      attribute interfaceEnabled;
+    ram      attribute lastNetworkingStatus;
+    ram      attribute lastNetworkID;
+    ram      attribute lastConnectErrorValue;
+    ram      attribute featureMap default = 1;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster DiagnosticLogs {
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 1;
+  }
+
+  server cluster GeneralDiagnostics {
+    emits event BootReason;
+    callback attribute networkInterfaces;
+    callback attribute rebootCount default = 0x0000;
+    callback attribute upTime default = 0x0000000000000000;
+    callback attribute totalOperationalHours default = 0x00000000;
+    callback attribute bootReason;
+    callback attribute activeHardwareFaults;
+    callback attribute activeRadioFaults;
+    callback attribute activeNetworkFaults;
+    callback attribute testEventTriggersEnabled default = false;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster Switch {
+    ram      attribute numberOfPositions default = 2;
+    ram      attribute currentPosition;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 1;
+  }
+
+  server cluster AdministratorCommissioning {
+    callback attribute windowStatus default = 0;
+    callback attribute adminFabricIndex default = 1;
+    callback attribute adminVendorId default = 0;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster OperationalCredentials {
+    callback attribute NOCs;
+    callback attribute fabrics;
+    callback attribute supportedFabrics;
+    callback attribute commissionedFabrics;
+    callback attribute trustedRootCertificates;
+    callback attribute currentFabricIndex;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster GroupKeyManagement {
+    callback attribute groupKeyMap;
+    callback attribute groupTable;
+    callback attribute maxGroupsPerFabric;
+    callback attribute maxGroupKeysPerFabric;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 1;
+  }
+}
+endpoint 1 {
+  device type ma_robotic_vacuum_cleaner = 116, version 1;
+
+
+  server cluster Identify {
+    ram      attribute identifyTime default = 0x0;
+    ram      attribute identifyType default = 0x0;
+    callback attribute generatedCommandList default = 0;
+    callback attribute acceptedCommandList default = 0;
+    callback attribute eventList;
+    callback attribute attributeList default = 0;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 2;
+  }
+
+  server cluster Groups {
```

Comment:
Robot vacuum device type does not have a Groups server in the spec....

Reply:
Correct. The device type in zap has these enabled. I will remove them from this example but I wonder if we should also remove them from the zap device type default.

Reply:
The XML looks OK for the RVC device type but zap adds the sceens and groups clusters when selecting this device type. Not sure why.

Reply:
That is very strange @hicklin. I just tried to open a brand-new ZAP file, create a new endpoint, select the "Matter Robotic Vacuum Cleaner (0x0074)" device type, and the only clusters that got default-enabled were Identify, Descriptor, RVC Run Mode, and RVC Operational State.

What steps did you do to see the behavior you saw?

Reply:
I did the same thing except that I opened the `template.zap` file instead of a brand-new ZAP file. I just tried it again and it still does it. When you open the `template.zap` there already is an device on  endpoint 1. I change it to the Matter Robotic Vacuum Cleaner and the Groups and Scenes clusters are automatically enabled. I also removed my `.zap` dir and tried again.

I also tried to create this configuration from a new ZAP file but I cannot even get the clusters to show up. Even adding an endpoint with the device profile was difficult. ZAP kept not populating the list of devices when adding a new endpoint in a new configuration.

Reply:
Please file ZAP issues at https://github.com/project-chip/zap/issues as needed!

Reply:
I have submitted [this issue](https://github.com/project-chip/zap/issues/1125).